Packs/Gurucul/ReleaseNotes/1_1_1.md

```diff
@@ -0,0 +1,46 @@
+#### Incident Fields
+- Updating Gra Case Status
+- Added Gra Anomalies
+- Added Gra Anomaly Name
+- Added Gra Anomaly Resource Name
+- Added Gra Anomaly Risk Score
+- Added Gra Anomaly Status
+- Added Gra Assignee
+- Added Gra Assignee Type
+- Added Gra Risk Accepted Date
+
+#### Integrations
+##### Gurucul-GRA
+
+- **New drop-down** ***Fetch Incident Cases*** **is added in** ***Add instance***. **The valid options are:**
+  - ***Case Per Anomaly***: Incident is created per Anomaly. This is default value set.
```

Comment:
please note i've requested twice to mention the breaking change here, as it is important for users to know that this updated modifies the behavior they had up until now
for example of breaking change changelog see https://github.com/demisto/content/blob/master/Packs/EWS/ReleaseNotes/1_8_2.md

Reply:
As per suggestion, we added a breaking change.